temoa/components/time.py

```
@@ -214,7 +219,9 @@ def loop_period_next_timeslice(M: 'TemoaModel', p, s, d) -> tuple[str, str]:
     return s_next, d_next
 
 
-def loop_season_next_timeslice(M: 'TemoaModel', p, s, d) -> tuple[str, str]:
+def loop_season_next_timeslice(
+    M: 'TemoaModel', p: 'Period', s: 'Season', d: 'TimeOfDay'
```

Ruff (0.14.0)

Unused function argument: `p`

(ARG001)